src/infobubble.js

```diff
@@ -1735,7 +1759,7 @@ InfoBubble.prototype.figureOutSize_ = function() {
   }
 
   this.contentContainer_.style['width'] = this.px(width);
-  this.contentContainer_.style['height'] = this.px(height);
+ // this.contentContainer_.style['height'] = this.px(height);
```

Comment:
Sorry, I had an issue with varied height content. If I set MaxHeight, for example, 400px, It applies it as css height:400px,  which is not ideal. It means even if content is shorter, it will make infoBubble longer. If content is longer than it adds scrollbar. Both of them are not acceptable. So, I just removed the height thing from my script. 
It may not be the issue for others, You can ignore that particular line change.